hydra/app/jobs/generate_image_thumbs_job.rb

```diff
@@ -3,42 +3,25 @@ class GenerateImageThumbsJob < ApplicationJob
 
   queue_as :import
 
-  def perform(identifier)
+  def perform(id, download_path)
+    return unless File.exist? download_path
+
     # find record
-    record = Acda.where(identifier: identifier).first
+    record = Acda.find(id)
```

Comment:
This is a direct Fedora find and is by nature slower.  Is that intentional?

Reply:
i did not know it was slower, so instead do a `Acda.where(id: id).first`?